Bottom: d7f1bf094bd0098a323290b0b3d74890801176e3
Top:    cd3bc7da493e1d5d7163e2b22ef19bc7b9e51325
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-02-28 16:26:56 +0800

Refresh of refresh-temp-0

---

diff --git a/drivers/net/virtio_net.c b/drivers/net/virtio_net.c
index a2571f9..9bbf35f 100644
--- a/drivers/net/virtio_net.c
+++ b/drivers/net/virtio_net.c
@@ -494,26 +494,6 @@ static struct sk_buff *virtnet_skb_xdp(struct receive_queue *rq,
 	return skb;
 }
 
-static struct sk_buff *virtnet_skb_xdp(struct receive_queue *rq,
-				       struct sk_buff *skb)
-{
-	struct bpf_prog *xdp_prog;
-	int ret;
-
-	rcu_read_lock();
-	xdp_prog = rcu_dereference(rq->xdp_prog);
-	if (xdp_prog) {
-		ret = do_xdp_generic(xdp_prog, skb);
-		if (ret != XDP_PASS) {
-			rcu_read_unlock();
-			return NULL;
-		}
-	}
-	rcu_read_unlock();
-
-	return skb;
-}
-
 static struct sk_buff *receive_small(struct net_device *dev,
 				     struct virtnet_info *vi,
 				     struct receive_queue *rq,
